robotiq_description/urdf/robotiq_gripper.ros2_control.xacro

```diff
@@ -1,21 +1,106 @@
 <?xml version="1.0"?>
 <robot xmlns:xacro="http://www.ros.org/wiki/xacro">
 
-    <xacro:macro name="robotiq_gripper_ros2_control" params="name com_port">
-        <ros2_control name="${name}" type="actuator">
+    <xacro:macro name="robotiq_gripper_ros2_control" params="
+        name
+        prefix
+        sim_ignition:=false
+        sim_isaac:=false
+        use_fake_hardware:=false
+        fake_sensor_commands:=false
+        com_port:=/dev/ttyUSB0">
```

Comment:
FYI I defaulted all of these args so the user only has to specify the one's they need. For example to load a config for Isaac...
```
   <xacro:robotiq_gripper name="RobotiqGripperSystem" prefix="" parent="gripper_mount_link" sim_isaac="true">
      <origin xyz="0 0 0" rpy="0 0 0" />
    </xacro:robotiq_gripper>
```

Reply:
Sounds good to me!